src/astronomer/flask_appbuilder/security.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2019 Astronomer Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from logging import getLogger
+
+from flask import request, abort
+from flask_appbuilder.security.manager import AUTH_REMOTE_USER
+from flask_appbuilder.security.views import AuthView
+from flask_login import login_user, current_user
+from jwcrypto import jwt, jwk, jws
+from jwcrypto.common import JWException
+
+
+log = getLogger(__name__)
+
+try:
+    from airflow.www_rbac.security import AirflowSecurityManager, EXISTING_ROLES
+except ImportError:
+    from airflow.www.security import AirflowSecurityManager, EXISTING_ROLES
+
+
+class AstroSecurityManagerMixin(object):
+    """
+
```

Comment:
let's remove empty new lines here.

Reply:
Oh yes, that was a placeholder for me to add docs :)

Reply:
Which I nave now added